channels/wager

```diff
@@ -2,8 +2,10 @@
 
 You can decide to either:
 
-1. Kill someone, and sell your soul in the process
+1. Kill someone, but sell your soul in the process
 
-2. Check someone's role, but the Devil will also see the result.
+2. Protect someone, but sell your soul in the process
```

Comment:
since when hell team gets attacked, it uses a soul for every attack but wager protecting doesnt?. Imo it feel a bit weird but i get that otherwise that option might not seem as interesting. 

Soul info: "If they are attacked several times a soul is used for every attack. "

Reply:
That's how it always was.